src/index.js

```diff
@@ -0,0 +1,27 @@
+import './styles/main.scss';
+import Tasks from './modules/ClassStore.js';
+
+const addForm = document.querySelector('#add-task');
+
+const tasksCollection = new Tasks();
+tasksCollection.displayTasks();
+
+addForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const inputTaskValue = document.querySelector('#input-add-task').value;
+  if (inputTaskValue === null || inputTaskValue === '') return;
+  tasksCollection.addTask(inputTaskValue);
+  document.querySelector('#input-add-task').value = '';
+});
+
+document.querySelector('.reflesh').addEventListener('click', () => {
+  tasksCollection.tasksArray = [];
+  localStorage.setItem('tasks', JSON.stringify(tasksCollection.tasksArray));
+  tasksCollection.displayTasks();
+});
+
+document.querySelector('.clear-completed').addEventListener('click', () => {
+  tasksCollection.tasksArray = tasksCollection.tasksArray.filter((task) => !task.complete);
+  localStorage.setItem('tasks', JSON.stringify(tasksCollection.tasksArray));
```

Comment:
You can use a function to update your local storage like `updateLocalStorage()` and call this function instead of typing out the long `setItem()` function.
```
const updateLocalStorage(tasksArray) {
    localStorage.setItem('tasks', JSON.stringify(tasksCollection.tasksArray));
}
```

Reply:
@tkz96 thanks :+1:  for this, I didn't think about that and it was costing me time writing the `setItem` function over and over again :smile: :handshake: